gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowExecutionResource.java

```diff
@@ -52,10 +55,19 @@
 public class FlowExecutionResource extends ComplexKeyResourceTemplate<FlowStatusId, EmptyRecord, FlowExecution> {
   private static final Logger LOG = LoggerFactory.getLogger(FlowExecutionResource.class);
   public static final String FLOW_STATUS_GENERATOR_INJECT_NAME = "FlowStatusGenerator";
+  public static final String EVENTBUS_INJECT_NAME = "EventBus";
+  public static final String HELIX_MANAGER_INJECT_NAME = "HelixManager";
+  public static final String FORCE_LEADER_INJECT_NAME = "ForceLeader";
   public static final String MESSAGE_SEPARATOR = ", ";
 
   @Inject @javax.inject.Inject @javax.inject.Named(FLOW_STATUS_GENERATOR_INJECT_NAME)
   FlowStatusGenerator _flowStatusGenerator;
+  @Inject @javax.inject.Inject @javax.inject.Named(EVENTBUS_INJECT_NAME)
+  EventBus eventBus;
+  @Inject @javax.inject.Inject @javax.inject.Named(HELIX_MANAGER_INJECT_NAME)
```

Comment:
It seems like we don't really need a HelixManager instance here and the dependency on helix module can be removed. Can HelixLeaderUtils create a singleton HelixManager instance that it uses to check the leader status?